src/Microsoft.OpenApi.OData.Reader/Operation/EntitySetPostOperationHandler.cs

```diff
@@ -159,5 +104,76 @@ protected override void AppendCustomParameters(OpenApiOperation operation)
                 AppendCustomParameters(operation, insert.CustomHeaders, ParameterLocation.Header);
             }
         }
+
+        /// <summary>
+        /// Get the entity content description.
+        /// </summary>
+        /// <returns>The entity content description.</returns>
+        private IDictionary<string, OpenApiMediaType> GetContentDescription()
+        {
+            OpenApiSchema schema = GetEntitySchema();
+            var content = new Dictionary<string, OpenApiMediaType>();
+
+            if (EntitySet.EntityType().HasStream)
+            {
+                var mediaTypes = Context.Model.GetCollection(EntitySet,
```

Comment:
`AcceptableMediaTypes` can not be applied to `EnitySet` or `Signleton`, it can only be applied to `EntityType`, `Property` or `Term` - https://github.com/oasis-tcs/odata-vocabularies/blob/61542d91d01a8bade005317d922270011a89a0a1/vocabularies/Org.OData.Core.V1.xml#L389.
> Lists the MIME types acceptable for the annotated entity type marked with HasStream="true" or the annotated binary, stream, or string property or term

Reply:
Thanks. 
Resolved